src/mlpack/methods/random_forest/random_forest.hpp

```diff
@@ -369,10 +383,14 @@ class RandomForest
                const size_t minimumLeafSize,
                const double minimumGainSplit,
                const size_t maximumDepth,
-               DimensionSelectionType& dimensionSelector);
+               DimensionSelectionType& dimensionSelector,
+               bool warmStart = false);
 
   //! The trees in the forest.
   std::vector<DecisionTreeType> trees;
+
+  //! The average gain of the forest.
+  double avgGain = 0.0;
```

Comment:
Can you initialize this in the constructor?  I know it is a bit more code, but every other class in mlpack uses initialization lists, so the consistency would be nice. :+1: